src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/config/StorageConfiguration.java

```diff
@@ -23,13 +26,22 @@ public StorageSharedKeyCredential getStorageSharedKeyCredential(
     // not used as needs test context so it can actually be build
     @Bean({
         "storage-client",
-        "bulkscan-storage-client",
-        "crime-storage-client"
+        "bulkscan-storage-client"
     })
     public BlobServiceClient getStorageClient(StorageSharedKeyCredential credentials) {
         return new BlobServiceClientBuilder().credential(credentials).buildClient();
     }
 
+    @Bean("crime-storage-client")
+    public BlobContainerClient getCrimeContainerClient() {
+        return mock(BlobContainerClient.class);
+    }
```

Comment:
perhaps bean should be renamed to `crime-container-client`?

Reply:
Yes, that's a better name. I considered changing it, but it requires config changes and adds a bit of risk time-wise. I'll create another PR with this change if you're fine with it.

Reply:
fair enough